doc/project-layout-specification.rst

```diff
@@ -182,7 +182,8 @@ Additionally, they can contains the following stanzas.
 name
 ----
 
-Sets the name of the project:
+Sets the name of the project. This is used by [`dune subst`](usage.html#dune-subst)
```

Comment:
I believe the syntax is:

```suggestion
Sets the name of the project. This is used by :ref:`dune subst <dune-subst>`
```

and then you need to add a line `.. _dune-subst`: before the `dune subst` section.